### ISSUE TITLE
"Failed loading configuration file" error with Psych 4

### ISSUE THREAD

**Issue by ybiquitous** [scoutapp/scout_apm_ruby#421]
Hello, I encounter the following error with Psych 4 and Rails 6.1.4:

```console
$ bin/rails r 'puts Rails.version'
[Scout] [06/30/21 17:26:59 +0900 F200420NT01.local (380)] INFO : Failed loading configuration file (/tmp/repro/config/scout_apm.yml): Unknown alias: defaults. ScoutAPM will continue starting with configuration from ENV and defaults
6.1.4
```

Here is a part of `Gemfile`:

```ruby
gem 'scout_apm', '4.1.1'
gem 'psych', '4.0.1'
```

Here is `config/scout_apm.yml`:

```yaml
common: &defaults
  name: YOUR_APPLICATION_NAME
  key: YOUR_APPLICATION_KEY
  monitor: true

test:
  monitor: false

development:
  <<: *defaults

production:
  <<: *defaults
```

When using Psych 3, this error never occurs:

```diff
-gem 'psych', '4.0.1'
+gem 'psych', '< 4'
```

I think this is a similar problem with rails/rails#42257 (Psych 4 incompatibility).

**Comment by dlanderson** [comment]
Resolved by #435 - pending release

**Comment by ybiquitous** [comment]
@dlanderson Thank you!
